tests/qchem/openfermion_pyscf_tests/test_convert_openfermion.py

```diff
@@ -327,32 +309,6 @@ def test_not_xyz(self, op):
         with pytest.raises(ValueError, match=_match):
             qml.to_openfermion(qml.to_openfermion(pl_op))
 
-    INVALID_OPS_WIRES = (
-        (
-            qml.ops.LinearCombination(
-                np.array([0.1, 0.2]),
-                [
-                    qml.operation.Tensor(qml.PauliX(wires=["w0"])),
-                    qml.operation.Tensor(qml.PauliY(wires=["w0"]), qml.PauliZ(wires=["w1"])),
-                ],
-            )
-        ),
-        ((qml.fermi.FermiWord({(0, 0): "+", (1, 1): "-"}))),
-    )
-
-    @pytest.mark.parametrize("pl_op", INVALID_OPS_WIRES)
-    def test_wires_not_covered(self, pl_op):
```

Comment:
Removed wire mapping for fermionic objects. It is problematic because we do not have support for flexible orbital labelling in FermiWord and FermiSentence objects.